### PR TITLE
Update Dockerfile to fix DocumentRoot config file

### PR DESCRIPTION
**Pull request by texai**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ RUN apt-get update \
  && apt-get install -y git zlib1g-dev \
  && docker-php-ext-install zip \
  && a2enmod rewrite \
- && sed -i 's!/var/www/html!/var/www/public!g' /etc/apache2/apache2.conf \
+ && sed -i 's!/var/www/html!/var/www/public!g' /etc/apache2/sites-enabled/000-default.conf \
```
Comment (adamlundrigan) [inline Dockerfile]: Is it sufficient to just update the default vhost, or should we continue to also update `apache2.conf`?

**Comment by rbroen** [comment]
Push, push, push. This change came just in time for me (after trying to solve it myself).
Thank you guys!

File: Dockerfile (hunk 1 above)
Comment (texai) [inline Dockerfile]: by default `apache2.conf` includes every `sites-enabled/*.conf` file.
So, you don't require edit your `apache2.conf` file

**Comment by bitwombat** [comment]
Just wasted 30 minutes tracking this same problem down :(  Accept please!

**Comment by webimpress** [comment]
Duplicate #361. For more information please check my comments:
https://github.com/zendframework/ZendSkeletonApplication/pull/361#issuecomment-262753255
https://github.com/zfcampus/zf-apigility-skeleton/pull/150#issuecomment-262752400

**Pull request closed without merging**